.github/workflows/renovate-bot.yml

```diff
@@ -0,0 +1,32 @@
+name: Renovate
+
+on:
+    schedule:
+      - cron: '0 6 * * *'
+    workflow_dispatch:
+
+jobs:
+  renovate:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/renovatebot/renovate
+      options: --user root # workaround for checkout
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4 # using checkout to access workspace
+      - name: Run Renovate
+        env:
+          RENOVATE_TOKEN: ${{ secrets.RENOVATE_TOKEN }}
+          RENOVATE_CONFIG_FILE: ${{ github.workspace }}/config.js
```

Comment:
Maybe we can rename this file to `renovate-config.js`, so people won't missunderstood it for some other general config file?

Reply:
Yes, I agree.